conf/machine/generic-armel-iproc.conf

```diff
@@ -6,7 +6,7 @@
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-yocto-onl"
 PREFERRED_VERSION_linux-yocto-onl ?= "6.1%"
 
-KMACHINE:onl-armel-iproc = "armel-iproc"
```

Comment:
Isn't there the same thing to fix on line 2 of this file? Maybe change that to something like "generic armel iProc ONL image".